packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -95,7 +94,7 @@ export default function Sidebar({
     return <></>;
   }
 
-  if (isNewMobileLayout && isTablet) {
+  if (!isLaptop && isTablet) {
```

Comment:
We don't even need the laptop condition here.
```suggestion
  if (isTablet) {
```

Reply:
Why not, isTablet stays true even in laptop sizes... 😞

Reply:
Apparently, in our JS it is not the case 😓 

Yeah, they kinda work differently from what we have in CSS. Hopefully we would be able to make them the same in the future.

Reply:
I think they work exactly like what we have in css, when we say `tablet:...` we mean everything tablet and above, unless it's overriten by any `laptop:...`, which I still don't like, I think it's not bad to maintain even breakpoints with min & max values so u don't have to write so many conditions when u want something only on a specific breakpoint.